database/schema.sql

```diff
@@ -2,6 +2,11 @@ CREATE DATABASE csr_lookup;
 
 \connect csr_lookup;
 
```

Comment:
We can add this line before creating the table so it gets dropped before creating.
DROP TABLE IF EXISTS table_name;

Reply:
Good idea, I just added it. Only have to make sure we don't run it by accident 😆

Reply:
No bro, I mean for each table, do you want to drop the whole database and recreate it?
Sounds unusual to me but why not!
But you will need to edit the line to:
DROP **DATABASE**  IF EXISTS  csr_lookup

Reply:
Oh man :man_facepalming: now it should actually work. I think it makes sense to drop the whole db to make sure you're really starting with a clean slate.

Reply:
Awesome!